.meta/sinks/gcp_stackdriver_logging.toml

```diff
@@ -0,0 +1,97 @@
+[sinks.gcp_stackdriver_logging]
+title = "GCP Stackdriver Logging"
+batch_max_size = 5242880
+batch_timeout_secs = 1
+beta = true
+buffer = true
+common = true
+delivery_guarantee = "best_effort"
+function_category = "transmit"
+healthcheck = true
+egress_method = "batching"
+input_types = ["log"]
+request_rate_limit_duration_secs = 1
+request_rate_limit_num = 100
+request_retry_attempts = 9223372036854775807
```

Comment:
Where is this number coming from? This seems like a lot of attempts 😄

Reply:
It's the maximum `usize` (ie 64-bit max). Most of the batched sinks use that number.

Reply:
Hm, this doesn't seem to be consistent. For example, the `aws_s3` sink says `5` for this value:

https://vector.dev/docs/reference/sinks/aws_s3/#retry_attempts

Should we open a separate PR to update these?

Reply:
Updating the request defaults for sinks other than this one would fall into its own issue/PR, yes.